tests/boardfarm_plugins/boardfarm_prplmesh/devices/prplmesh_compose.py

```diff
@@ -0,0 +1,100 @@
+# SPDX-License-Identifier: BSD-2-Clause-Patent
+# SPDX-FileCopyrightText: 2020 the prplMesh contributors (see AUTHORS.md)
+# This code is subject to the terms of the BSD+Patent license.
+# See LICENSE file for more details.
+
+import os
+import time
+
+import boardfarm
+from environment import ALEntityDocker, _get_bridge_interface
+from .prplmesh_base import PrplMeshBase
+from sniffer import Sniffer
+
+rootdir = os.path.abspath(os.path.join(os.path.dirname(__file__), '../../../..'))
+
+
+class PrplMeshCompose(PrplMeshBase):
```

Comment:
I don't think we should define a new class. Instead, change the PrplMeshDocker class to use compose instead of naked docker.